<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qt&amp;&amp;theFunc(bunch) Pixel Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="Canvas" name="canvas" native="true">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>0</y>
      <width>512</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Tools" name="tools" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>194</width>
      <height>512</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Preview" name="preview" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>194</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="Palette" name="palette" native="true">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>256</y>
      <width>194</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="FramesView" name="framesView" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>512</y>
      <width>900</width>
      <height>188</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <zorder>canvas</zorder>
   <zorder>preview</zorder>
   <zorder>palette</zorder>
   <zorder>framesView</zorder>
   <zorder>tools</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport">
     <property name="title">
      <string>Export</string>
     </property>
     <addaction name="actionPNG"/>
     <addaction name="actionJPG"/>
     <addaction name="actionGIF"/>
     <addaction name="actionTileset"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
    <addaction name="menuExport"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionToggle_Gridlines"/>
    <addaction name="actionToggle_Overlay"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionImport">
   <property name="text">
    <string>Import ...</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionPNG">
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionJPG">
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="actionGIF">
   <property name="text">
    <string>GIF</string>
   </property>
  </action>
  <action name="actionTileset">
   <property name="text">
    <string>Tileset</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionToggle_Gridlines">
   <property name="text">
    <string>Toggle Gridlines</string>
   </property>
  </action>
  <action name="actionToggle_Overlay">
   <property name="text">
    <string>Toggle Overlay</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Preview</class>
   <extends>QWidget</extends>
   <header>preview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Palette</class>
   <extends>QWidget</extends>
   <header>palette.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Tools</class>
   <extends>QWidget</extends>
   <header>tools.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FramesView</class>
   <extends>QWidget</extends>
   <header>framesview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
